### PR TITLE
Change Hyperdash in Platters rule (osu!catch)

### PR DESCRIPTION
**Pull request by O1CDeif**
Changes to the osu!catch Ranking Criteria after consensus with the community.

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ These terms are frequently appearing when getting in touch with osu!catch diffic
 -   **Hyperjump:** A more exaggerated spacing which cannot be caught by normal dashing. During play, hyperdash will be triggered between the two objects, characterised by a coloured outline on the first object.
 -   **Edge Dash:** A very large spacing between two objects where the required Trigger Distance is not reached, and as such, a hyperjump is not generated. The first object must be caught with the edge of the plate in order to catch the second object at all.
 -   **Trigger distance:** The minimum spacing between two objects at which a hyperdash is generated between them.
+-   **Equivalent BPM:** A BPM value in which a certain snap is time-equivalent (same duration in ms) to a reference BPM/snap pair. This is calculated as Equivalent BPM = (Reference BPM)*(Desired Snap)/(Reference Snap).
```
Comment (Loctav) [inline wiki/Ranking_Criteria/osu!catch/en.md]: What is a snap pair? Which BPM reference? Generally I read that 5 times and I still don't know what it is.
Also make the equation in code blocks (\`text\`)

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 2]
```diff
@@ -114,7 +115,8 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 
 #### Rules
 
--   **Basic hyperdashes (1/1, 1/2) may begin to appear, but must not be used in conjunction with each other.** Platters should serve as an introduction to hyperdashing, meaning strong hypers (e.g. 1.5x above the trigger distance) and hypers combined with antiflow patterns must not be used. 1/3 and 1/4 hyperdashes may be used singularly (not in conjunction with other dashes or hyperdashes) if the BPM is less than 120.
+-   **Basic hyperdashes (1/1, 1/2) may begin to appear, but must not be used in conjunction with each other.** Platters should serve as an introduction to hyperdashing, meaning strong hypers (e.g. 1.5x above the trigger distance) and hypers combined with antiflow patterns must not be used.
```
Comment (Loctav) [inline wiki/Ranking_Criteria/osu!catch/en.md]: Antiflow patterns? No definition present
You should mention that 1.5x refers to Distance Snap

File: wiki/Ranking_Criteria/osu!catch/en.md (hunk 1 above)
Comment (O1CDeif) [reply wiki/Ranking_Criteria/osu!catch/en.md]: It's meant to be a pair "reference BPM/reference snap". Changing the wording to make it more clear

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 3]
```diff
@@ -135,6 +138,7 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 #### Rules
 
 -   **More complex hyperdashes (1/3, 1/4) may begin to appear, but no type of hyperdash can be used between more than four consecutive objects.** Additionally, hyperdashes of different beat snap (1/2, 1/4, etc.) must not be used between consecutive objects.
+-   **1/6 hyperdashes may be used singularly (not in conjunction with other dashes or hyperdashes) if the BPM is less than 160.** Higher snap hyperdashes may be used singularly if the song BPM is below the equivalent BPM.
```
Comment (Loctav) [inline wiki/Ranking_Criteria/osu!catch/en.md]: See above. Must be changed here, too, obviously.

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 4]
```diff
@@ -28,6 +28,8 @@ These terms are frequently appearing when getting in touch with osu!catch diffic
 -   **Hyperjump:** A more exaggerated spacing which cannot be caught by normal dashing. During play, hyperdash will be triggered between the two objects, characterised by a coloured outline on the first object.
 -   **Edge Dash:** A very large spacing between two objects where the required Trigger Distance is not reached, and as such, a hyperjump is not generated. The first object must be caught with the edge of the plate in order to catch the second object at all.
 -   **Trigger distance:** The minimum spacing between two objects at which a hyperdash is generated between them.
+-   **Equivalent BPM:** A BPM value in which a certain snap is time-equivalent (same duration in ms) to a paired reference BPM/snap. This is calculated as Equivalent BPM = (`Reference BPM`)*(`Desired Snap`)/(`Reference Snap`).
```
Comment (Loctav) [inline wiki/Ranking_Criteria/osu!catch/en.md]: I am yet to figure out how this formula is even supposed to be used. Remove the entire thing and rather work with clean-cut millisecond threshold. This saves people a lot of headache.

**Pull request merged**